drivers/sennheiser/team_connect_ceiling_medium.cr

```diff
@@ -0,0 +1,99 @@
+require "placeos-driver"
+require "placeos-driver/interface/muteable"
+require "./models"
+
+class Sennheiser::TeamConnectCM < PlaceOS::Driver
+  include Interface::AudioMuteable
+  include Sennheiser::Models
+  descriptive_name "TeamConnect Ceiling Medium"
+  generic_name :TCCM
+
+  uri_base "https://device_ip"
+
+  default_settings({
+    username:      "api",
+    password:      "",
```

Comment:
```suggestion
    basic_auth: {
      username: "api",
      password: "",
    },
```

you don't actually need to handle auth in the headers, the driver will do it for you